### DISCUSSION TITLE
Display Rejected RSS Feed Suggestions in Dashboard to Avoid Duplicates

### DISCUSSION THREAD

**Discussion by Anmol-Baranwal** [dailydotdev/daily#843] [Feature Requests]
It would be great if we could see the suggestions that have been rejected or are waiting for approval in the dashboard. This would make the process easier and notify users of the list of rejected RSS feeds to avoid duplicate suggestions.

> I know the documentation clearly states that, other than email, there is no other way to do this

**Comment by idoshamun** [comment]
Hi @Anmol-Baranwal,

Thanks for the suggestion! :)
